<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChoiceToGraph</class>
 <widget class="QMainWindow" name="ChoiceToGraph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите количество вершин в графе</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spb_choice">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_continue">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Продолжить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_back">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>290</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
